zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -549,12 +554,21 @@ public static X509TrustManager createTrustManager(
             KeyStore ts = loadTrustStore(trustStoreLocation, trustStorePassword, trustStoreTypeProp);
             PKIXBuilderParameters pbParams = new PKIXBuilderParameters(ts, new X509CertSelector());
             if (crlEnabled || ocspEnabled) {
```

Comment:
```suggestion
            if (crlEnabled) {
```

I think we probably should make this consistent how jdk handle "com.sun.net.ssl.checkRevocation" and "ocsp.enable". Revocation checking will only be enabled through "com.sun.net.ssl.checkRevocation" but not "ocsp.enable".